app/server/models/user.py

```diff
@@ -39,6 +43,10 @@ class User(ManyOrgBase, ModelBase):
 
     first_name = db.Column(db.String())
     last_name = db.Column(db.String())
+    preferred_language = db.Column(db.String(), default='en_au')
```

Comment:
per what we discussed, we want to remove the default

Reply:
sharing context here -- turns out we don't want a default because USSD state machine sees if preferred language is set. if it's not, then it sends user through initial language configuration. so instead of a default preferred language, what would be better is having a default fallback language perhaps for sempo's system (could be en_AU) that we use if user has no preferred language

Reply:
that makes sense to me